docs/host.md

```diff
@@ -5,63 +5,68 @@
 
 In Geoweaver, `Host` means computing machines (`Linux/Windows/Mac computers`). It is where the code gets executed, data gets processed, and history is generated.
 
-In most time, `Host` is dedicated to be the local machine where Geoweaver is installed. Once Geoweaver is started, it will add a default Host `Localhost` which means the current machine Geoweaver is sitting on. For most people, `Localhost` is all they need. They can run all their processes on `Localhost`.
+Mostly, `Host` is dedicated to the local machine where Geoweaver is installed. Once Geoweaver is started, it will add a default Host `Localhost` which means the current machine Geoweaver is sitting on. For most people, `Localhost` is all they need. They can run all their processes on `Localhost`.
 
-For cool kids, they might want to run their processes on remote servers, e.g., AWS EC2 instances, HPC. In that case, Geoweaver provides convenient options to enroll remote servers and run your code there but manage all of these distributed work in one place. This is a life-saving feature for multiple-server users. No kidding.
+Geoweaver has an amazing feature that's a lifesaver for multiple-server users. It allows you to enroll these servers and run your code on them without the hassle of having to manage it all separately. AWS EC2 instances and HPC are perfect examples of servers you can easily integrate with Geoweaver, and you'll be able to benefit from its time-saving benefits.
 
-This tutorial will show how to enroll a new `Host` and make it ready for running processes in Geoweaver.
+  
+This tutorial will help how to enroll a new `Host` and make it ready for running processes in Geoweaver.
+ 
 
 ## Create a Host
 
-1. Click the plus button after `Hosts` on the right panel.
+1. Click the `New Host` button at the `Hosts` on the left panel.
+ 
+2. Leave the `Host Type` as default, add `Host Name`, `Host IP`, `Port`, and `User Name` of your server in the shown dialog box.
 
-2. Input `Host Name`, `Host IP`, `Port`, and `User Name` of your server in the shown dialog. Here is an example:
```

Comment:
Is the button name changed? 
from Host Type to Host Name in UI ?

Reply:
No there is another button as Host type with the different hosts, and the rest of the buttons are same.

Reply:
👍🏻